dev-games/ogre/ogre-1.10.12.ebuild

```diff
@@ -26,28 +32,25 @@ RDEPEND="
 	dev-libs/zziplib
 	media-libs/freetype:2
 	virtual/glu
-	virtual/opengl
+	opengl? ( virtual/opengl )
```

Comment:
as this is now optional, you should move this line to the other options below (and keep them sorted)

Reply:
Of course, you are right. I remember that this was the plan, but I do not know why I forgot to sort that one away...

...dammit ... it is even incomplete. Not only virtual/opengl, but also virtual/glu depend on the opengl USE flag.
=> fixed